src/views/EntryList.test.jsx

```diff
@@ -0,0 +1,27 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import { UserProvider } from '../context/userContext';
+
+import EntryList from './EntryList';
+
+describe('EntryList', () => {
+  it('Should render list of characters', async () => {
+    render(
+      <MemoryRouter>
+        <UserProvider>
+          <EntryList />
+        </UserProvider>
+      </MemoryRouter>
+    );
+    screen.getByText(/loading/i);
+
+    const heading = await screen.findByText('Entry List');
+    expect(heading).toBeInTheDocument();
+
+    screen.getByPlaceholderText('new entry');
+
+    screen.getAllByRole('list');
+
+    screen.debug();
+  });
```

Comment:
You will want to test the entryList within the app.test file, since entry list is a private route, we will have to enter entry list after signing up so that our user is set in context.